content/courses/program-optimization/rust-macros.md

```diff
@@ -207,33 +202,32 @@ of a `println!` macro call with the `LitStr` AST as its argument.
 
 Note that the `quote!` macro generates a `TokenStream` of type
 `proc_macro2::TokenStream`. To return this `TokenStream` to the Rust compiler,
-you need to use the `.into()` method to convert it to `proc_macro::TokenStream`.
-The Rust compiler will then use this `TokenStream` to replace the original proc
-macro call in the source code.
+use the `.into()` method to convert it to `proc_macro::TokenStream`. The Rust
```

Comment:
Just wanted to say I love the quality of your writing and attention to detail in your edits.